src/core/xconnection.rs

```diff
@@ -419,11 +474,11 @@ impl XcbConnection {
         };
 
         // https://www.mankier.com/3/xcb_intern_atom
-        let atoms: HashMap<&'static str, u32> = ATOMS
+        let atoms: HashMap<Atom, u32> = ATOMS
             .iter()
             .map(|atom| {
                 // false == always return the atom, even if exists already
-                let val = xcb::intern_atom(&conn, false, atom)
+                let val = xcb::intern_atom(&conn, false, (*atom).into())
```

Comment:
does changing line 479 to `.map(|&atom| {` allow you to simplify this to just `atom.into()` ?

(see [this playground link](https://play.rust-lang.org/?version=stable&mode=debug&edition=2018&gist=42cc9817ce85dcecdb768ea9a7932375) for an example of what I mean)

Reply:
That does look a bit nicer, thanks!